src/Tribe/PUE/Checker.php

```diff
@@ -488,47 +523,41 @@ private function get_api_message( $info ) {
+		 * Displays an error notice if a premium plugin is activated and the license is expired
+		 *
+		 * @since 4.3
+		 *
+		 * @return bool|string
 		 */
-		public function display_json_error() {
-			$pluginInfo       = $this->json_error;
-			$update_dismissed = $this->get_option( $this->dismiss_upgrade );
+		public function display_license_error_message() {
+			$plugin_info = $this->plugin_info;
 
-			$is_dismissed = ! empty( $update_dismissed ) && in_array( $pluginInfo->version, $update_dismissed ) ? true : false;
-
-			if ( $is_dismissed ) {
-				return;
+			if ( ! current_user_can( 'install_plugins' ) ) {
+				return false;
 			}
 
-			if ( ! current_user_can( 'administrator' ) ) {
-				return;
+			if ( ! isset( $plugin_info->api_invalid ) ) {
+				return false;
 			}
 
-			//only display messages if there is a new version of the plugin.
-			if ( version_compare( $pluginInfo->version, $this->get_installed_version(), '>' ) ) {
-				if ( empty( $pluginInfo->api_invalid ) || $pluginInfo->api_invalid != 1 ) {
-					return;
-				}
+			$expired_license_msg     = $this->get_api_message( $plugin_info );
+			$expired_license_message = str_replace( '%plugin_name%', '<strong>' . $this->get_plugin_name() . '</strong>', $expired_license_msg );
 
-				$msg = $this->get_api_message( $pluginInfo );
-
-				//Dismiss code idea below is obtained from the Gravity Forms Plugin by rocketgenius.com
-				?>
-				<div class="updated" style="padding:5px; position:relative;" id="pu_dashboard_message"><?php echo wp_kses( $msg, 'post' ); ?>
-					<a href="javascript:void(0);" onclick="PUDismissUpgrade();" style="float:right;">[X]</a>
-				</div>
-				<script type="text/javascript">
-					function PUDismissUpgrade() {
-						jQuery("#pu_dashboard_message").slideUp();
-						jQuery.post( ajaxurl, {
-							action: "<?php echo esc_attr( $this->dismiss_upgrade ); ?>",
-							version: "<?php echo esc_attr( $pluginInfo->version ); ?>",
-							cookie: encodeURIComponent(document.cookie)
-						} );
-					}
-				</script>
-				<?php
+			$html[] = '<img class="tribe-spirit-animal" src="' . esc_url( Tribe__Main::instance()->plugin_url . 'src/resources/images/spirit-animal.png' ) . '">';
+			$html[] = '<p>' . wp_kses( $expired_license_message, 'post' ) . '</p>';
+
+			if ( isset( $plugin_info->api_expired ) ) {
+				$expired_link   = sprintf( '<a href="http://m.tri.be/195d" target="_blank">%1$s <span class="screen-reader-text">%2$s</span></a>', esc_html__( 'Renew your license', 'tribe-common' ), esc_html__( '(opens in a new window)', 'tribe-common' ) );
+				$html[] = '<p>' . sprintf( __( '%s to get access to the latest versions including bug fixes, security updates, and new features.', 'tribe-common' ), $expired_link ) . '</p>';
+			} else {
+				$license_tab = admin_url( 'edit.php?page=tribe-common&tab=licenses&post_type=tribe_events' );
+				$license_tab_link = sprintf( '<a href="' . $license_tab . '">%s</a>', esc_html__( 'Add your license key', 'tribe-common' ) );
```

Comment:
Here and the preceding/following lines: might it be cleaner to ditch `sprintf()` and use string concatenation (as you are doing with `$license_tab` anyway?

We often use `sprintf()` and co where the string to be translated contains placeholders, but in this block things are inverted so there's no particular need.